libcudacxx/test/libcudacxx/std/containers/views/views.span/span.cons/stdarray.pass.cpp

```diff
@@ -62,7 +62,7 @@ __host__ __device__ void checkCV()
 }
 
 template <typename T, typename U>
-__host__ __device__ TEST_CONSTEXPR_CXX17 bool testConstructorArray()
+__host__ __device__ TEST_CONSTEXPR_CXX14 bool testConstructorArray()
```

Comment:
Question: why not just 
```suggestion
__host__ __device__ constexpr bool testConstructorArray()
```
The ctor of `span` is also unconditionally `constexpr`.